src.js

```diff
@@ -9,27 +9,34 @@ const COLORS = [
   '#00FFB4'  // teal
 ];
 const FLOWER_RADIUS = 14;
+const VIRUS_START_RADIUS = 50;
 
 function _$(element) {
 }
 
-function $(element) {
-  return new _$(element);
-}
-
 _$.prototype.ready = callback => {
   document.addEventListener("DOMContentLoaded", event => {
     callback();
-  }, false );
+  }, false);
+}
+
+function $(element) {
+  return new _$(element);
 }
 
 $(document).ready(() => { 
   const canvas = new Canvas();
   window.canvasReset = canvas.reset;
+  canvas.drawGrid();
+
+  var i;
+  for(i = 0; i < 5; i++){
```

Comment:
Add the `var` inside the for. Also, it would be better to use a `let`. And add a space after the `)`.

You can read more about the ES6 style [here](https://github.com/airbnb/javascript).

We should probably use their linter.